platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/GossipEvent.java

```diff
@@ -36,8 +37,21 @@
  */
 public class GossipEvent implements BaseEvent, ChatterEvent {
     private static final long CLASS_ID = 0xfe16b46795bfb8dcL;
-    private BaseEventHashedData hashedData;
```

Comment:
why don't we just make the signature part of this class and get rid of `BaseEventUnhashedData` & `BaseEventImpl `?

Reply:
The older serialization scheme has a version number of BaseEventUnhashedData present.  If I can remove it use in the new serialization scheme.  It's easier to keep the class until it is completely not used and then remove the class once the serialization scheme is not related to it.

Reply:
It's going to be a lot of work to remove the use of the class, touches a lot of code that I haven't touched yet.  If you want me to do it, I will, but I've made the new serialization scheme for GossipEvent without it.

Reply:
I think its ok to keep it for migration, but then depreciate it and remove it in a subsequent version